content/Arduino Cloud/General Cloud/Configure-the-network-credentials-of-a-Thing.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configure the network credentials of a Thing"
+id: 14416141314332
+---
+
+In this article:
+
+- [Configuring a Thing without preset network credentials](#without_preset)
+- [Configuring a Thing with preset network credentials](#preset)
+
+## Configuring a Thing without preset network credentials {#without_preset}
+
+1. Go to the [Things tab](https://app.arduino.cc/things) and click the "+Thing" button in the top right corner.
+
+    ![Thing button in the top right corner](img/thing_button.png)
+
+1. After creating a Thing, select your device by clicking the "Select Device" button in the "Associated Device" section.
+
+    ![associated device section with select device button](img/associated_device.png)
+
+1. After selecting your device, you can enter your network credentials.
+
+    ![network credential section with configure button](img/Network_add.png)
+
+    Click the "Configure" button to enter your network credentials.
+
+    - Wi-Fi: Enter your Wi-Fi name (SSID) and password (encryption key).
+    - Arduino SIM: Use these credentials.
+    - Arduino with LoRa®: The credentials are added automatically.
+    - ESP32 and ESP8266: Enter the secret key that was provided when adding the device.
```

Comment:
```suggestion
1. Go to the [Things tab](https://app.arduino.cc/things) and select a Thing, or create a new Thing by clicking on the **+ Create Thing** button or the **+Thing** button in the top right corner.

1. If the Thing doesn't have an associated device, go to the **Associated Device** section and click the **Select Device** button. Get help [adding a device to Arduino Cloud](https://support.arduino.cc/hc/en-us/articles/360016495559-Add-and-connect-a-device-to-Arduino-Cloud#upload-sketch) if needed.

    ![associated device section with select device button](img/associated_device.png)

1. Go to the **Network Credentials** section and click the **Configure** button.

    ![network credential section with configure button](img/Network_add.png)

1. Enter your network credentials: 
    - Wi-Fi: Enter your Wi-Fi name (SSID) and password (encryption key).
    - If using ESP32 and ESP8266: Enter the secret key that was provided when adding the device.
    - If using Arduino SIM: Use [these credentials](https://support.arduino.cc/hc/en-us/articles/360013825159-What-are-the-credentials-for-the-Arduino-SIM-card).
    - If using Arduino with LoRa®: The credentials are added automatically.
    
 1. Go to the **Sketch** tab in the top right and upload the sketch to load the credentials on the board.
```
1. Adding the case of users already having a Thing created (but without preset network credentials), or having to create a new Thing. 
2. Also adding the case of users already having an associated device.
3. Added the link to Arduino SIM's credentials. 
4. Added one extra step for entering the network credentials. Since a new window is open after clicking on the **Configure** button, the extra step can make it easier to follow. 
5. Added a set for uploading the sketch to the board

Reply:
where is the "+ Create Thing" button? Are you sure there is a button labeled like that? 😄

Reply:
![image](https://github.com/arduino/help-center-content/assets/83598143/22acb556-06af-46b1-aad4-fee1574f4b3e)

Ah yea, you're right. Thanks for clarifying. 😄